kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/KinAccount.java

```diff
@@ -137,6 +156,66 @@
     @NonNull
     Balance getBalanceSync() throws OperationFailedException;
 
+    /**
+     * Create {@link Request} for getting the current confirmed aggregated balance in kin
+     * <p> See {@link KinAccount#getAggregatedBalanceSync()} for possibles errors</p>
+     *
+     * @return {@code Request<Balance>} Balance - the aggregated balance in kin
+     */
+    @NonNull
+    Request<Balance> getAggregatedBalance();
+
+    /**
+     * Get the current confirmed aggregated balance in kin
```

Comment:
what happens if no controlled balance? worth noting here

Reply:
Added it could be be an empty list and this is what that will be retuned if no controlled accounts